modular_zubbers/code/modules/antagonists/bloodsucker/structures/objects.dm

```diff
@@ -228,12 +233,11 @@
 	custom_materials = list(/datum/material/silver = SHEET_MATERIAL_AMOUNT)
 	embed_data = /datum/embed_data/stake/silver
 	staketime = 15 SECONDS
-	kills_blodsuckers = TRUE
 
 /obj/item/stake/hardened/silver/examine_more(mob/user)
 	. = ..()
 	if(HAS_TRAIT(user.mind, TRAIT_BLOODSUCKER_HUNTER))
-		. += span_notice("You know that the [src] can cause a Final Death to a vile Bloodsucker if they are asleep or dead.")
+		. += span_notice("You know that the [src] can remo")
```

Comment:
you know that the [src] can remo. so real. I hope it can remo

Reply:
ima make this visible for anyone, so it's a bit more clear